### PR TITLE
Rev-suffix all assets

### PR DESCRIPTION
**Pull request by charmander**
Improves cacheability and can be made available to static file server without proxying.

Default thumbnails and avatars really need to be cleaned up, deciding on their rendering at the renderer level.

### CODE REVIEW

File: weasyl/templates/etc/site_update.html [hunk 1]
```diff
@@ -6,7 +6,7 @@ <h3>${update.title}</h3>
 
     <figure class="thumbnail thumbnail-left clear">
       <a class="avatar" href="/~${owner.login_name}" title="${owner.profile.username}">
-        <img src="${owner.avatar['display_url']}" alt="avatar of ${owner.profile.username}" />
+        <img src="${owner.avatar_display_url or resource_path('img/default-avatar.jpg')}" alt="avatar of ${owner.profile.username}" />
```
Comment (sl1-1) [inline weasyl/templates/etc/site_update.html]: More logic in templates :(
Will add looking into moving this logic back to code in the new templates

File: static/google5586570e8454bded.html [hunk 2]
```diff
@@ -1 +0,0 @@
-boop
```
[the remark this answers is not in the document]
Comment (charmander) [reply static/google5586570e8454bded.html]: It is a site verification file, but there’s already a meta tag. I guess we should confirm which one is used. Maybe both are.

**Comment by kfkitsune** [comment]
>     * I have not yet seen if this functions as expected in the vagrant environment.
```
kyra@kitty ~/g/weasyl> make guest-run
vagrant ssh -c 'cd weasyl && make run'
cp -n config/site.config.txt.example config/site.config.txt
cp -n config/weasyl-staff.example.py config/weasyl-staff.py
node build.js
/vagrant/build.js:94
const copyStaticFiles = async (relativePath, touch) => {
                              ^

SyntaxError: Unexpected token (
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:549:28)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.runMain (module.js:611:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:160:9)
Makefile:86: recipe for target 'build/rev-manifest.json' failed
make: *** [build/rev-manifest.json] Error 1
Connection to 127.0.0.1 closed.
Makefile:112: recipe for target 'guest-run' failed
make: *** [guest-run] Error 2

vagrant@vagrant-weasyl:~$ node --version
v6.17.1
```
I wonder if this error is due to an (even older(!!)) version of ``nodejs`` than Travis had [(8.17.0 it looks like)](https://travis-ci.org/Weasyl/weasyl/jobs/648737259#L257), which caused the CI build to break. Error is after a ``distclean`` on weasyl/libweasyl, and ``vagrant provision``.

Current/active nodejs LTS is v12, it looks like.

**Comment by kfkitsune** [comment]
Vagrant now works as expected after the synchronization of the Docker changes to ``Vagrantfile``. And unless I am missing something, it seems this is being used on prod as per the short-SHA. The remaining question is the Google verification file. (Though if that's for like, the Google Webmaster Console (or whatever), if I was reading the support pages right, we can just recreate it if needed, so... it shouldn't break anything by getting rid of it? At least, that can't be restored when needed.)

**Comment by charmander** [comment]
> The remaining question is the Google verification file. (Though if that's for like, the Google Webmaster Console (or whatever), if I was reading the support pages right, we can just recreate it if needed, so... it shouldn't break anything by getting rid of it? At least, that can't be restored when needed.)

Yep, the support page says it’s usually fine to interrupt verification. Ikani’ll be able to confirm whether we need to restore something, and where. (The static file approach *is* better, so we can get rid of the meta tag at that time too.)

**Pull request merged**